packages/web-components/fast-foundation/src/radio/radio.ts

```diff
@@ -101,17 +101,20 @@ export class Radio extends FormAssociated<HTMLInputElement> implements RadioCont
     @observable
     public checked: boolean = this.defaultChecked;
     private checkedChanged(): void {
-        if (!this.dirtyChecked) {
-            this.dirtyChecked = true;
-        }
+        // TODO: temporarily removed as it's causing issues with
+        // changing the value via code and from radio-group
+        // if (!this.dirtyChecked) {
+        //     this.dirtyChecked = true;
+        // }
```

Comment:
Is this still temporarily removed? If so, can we have something tracking it?

Reply:
nope, added it back in now